packages/core/src/external/carequality/ihe-gateway-v2/utils.ts

```diff
@@ -11,3 +12,17 @@ export function extractText(textOrTextObject: TextOrTextObject): string {
   }
   return String(textOrTextObject);
 }
+
+export function getSlotValue(slot: Slot | undefined): string | undefined {
+  if (!slot) {
+    return undefined;
+  }
+  if (typeof slot.ValueList === "object" && slot.ValueList !== undefined) {
+    const value = slot.ValueList.Value;
```

Comment:
Can't value be `undefined`?

Reply:
yes but then it wont be array. adding an explicit check though

Reply:
My point is that it would return the string "undefined", which is not great as it might make its way into the output and does not match the function's contract to return `undefined` when the value doesn't exist/can't be determined (AFAIK).

I'd add this before the check for `isArray`:

```ts
if (!value) return undefined;
```

Reply:
understood and fixed!